config/environments/development.rb

```diff
@@ -14,8 +14,16 @@
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
-  config.action_mailer.default_url_options = { host: 'rubygamedev.dev' }
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+  address: "localhost",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.